packages/cwp-template-full/index.js

```diff
@@ -22,6 +23,10 @@ module.exports = async ({ appName, root }) => {
         );
     }
 
+    //Commit .gitignore
+    execa.sync("git", ["add", ".gitignore"], { cwd: root });
+    execa.sync("git", ["commit", "-m", `"Commited .gitignore"`], { cwd: root });
```

Comment:
Please update the commit message to follow conventional commit standard. `chore: initialize git`.
`Commited` is a past tense, it's not the standard.